sdcm/tester.py

```diff
@@ -935,9 +932,8 @@ def _nodetool_repair(node):
                              num_retry_on_failure=3)
         self.log.debug("system_auth description: %s", res.stdout)
         self.log.info('repair system_auth keyspace ...')
-        parallel_objects = ParallelObject(self.db_cluster.nodes, num_workers=min(
-            32, len(self.db_cluster.nodes)), timeout=MINUTE_IN_SEC * 25)
-        parallel_objects.run(_nodetool_repair)
+        for node in self.db_cluster.nodes:
+            node.run_nodetool(sub_cmd="repair -pr system_auth", timeout=MINUTE_IN_SEC * 20)
```

Comment:
IIUC the flag `-pr` is only on primary replica, hence we ran them in parallel... so, shouldn't we remove the `-pr` at all if we are running sequentially?

Reply:
`-pr` is recommended according to https://github.com/scylladb/scylla-enterprise/issues/3103#issuecomment-1633394228

Reply:
`-pr` is recommended when running all nodes in parallel, and you are now calling them sequentially, and it may also be unnecessary, as it would work doing only once, in one of the nodes without the `-pr`

IIUC @asias ' comment, the test currently does the right thing, but better to hear a confirmation